api-guidelines/global/core-principles/documentation.md

```diff
@@ -1,57 +1,39 @@
+This includes, but is not limited to:
 
-:::: accordions
-::: accordion Clear structure
-**Usability**
+- [Accessibility and usability](#accessibility-and-usability)
+- [Code examples](#code-examples)
+- [Context and walkthroughs](#context-and-walkthroughs)
+- [Up-to-dateness](#up-to-dateness)
 
-- A user-friendly navigation makes it easy to discover the API and find all relevant information.
-- The sidebar is cleary structured and easy to navigate.
-- API endpoints and examples are shown in context.
-
-**Getting started**
-
-- Basic information is available at one central place, such as info about HTTP methods, API operations, request/response headers, versioning, pagination, or errors.
-- A short and simple guide provides an overview of the most likely tasks to perform with the API.
-- Users can get in touch with us via a contact form.
-
-**Tutorials**  
-With step-by-step walkthroughs we'll cover specific functionality that can be implemented with the API.
-
-**Familiarize users with the API**  
-We'll cater for experienced users that already have an idea of the endpoints to be used, and at the same time provide the respective structure and information for inexperienced users who need a thorough introduction and/or example use cases.
+## Accessibility and usability
 
-**Discoverable and easy to digest**
-
-- All information is easily discoverable, easy to digest, and prepared in a way that the user can effectively work with it.
+- We provide access to all API endpoints in a central location, some sort of documentation portal at the best, to ensure that users can easily use the API.
+- The documentation is browsable.
+- A well-structured sidebar ensures that the documentation can be navigated so that all the endpoints can be accessed.
+- We give users the chance to get in touch with us.
+- All information is easily discoverable, easy to digest, and prepared in a way that users can effectively work with it.
 - We'll be consistent with our tone of voice, terminology, attribute names, API endpoint design, requests and responses, and counting.
-- The layout supports accordingly, e.g. with syntax highlighting or multi-column layout.
-  :::
+- The layout supports usability accordingly, e.g. with syntax highlighting or multi-column layout.
 
-::: accordion Code examples
-**Default values**  
-Request and response examples that belong to every endpoint documentation have meaningful default values that follow a specific story.
+## Code examples
 
-**Code examples and schemas**  
-Every endpoint documentation comes with examples and also provides schemas listing the available attributes with explanatory text.
+- Every endpoint documentation comes with examples and also provides schemas listing the available attributes with explanatory text.
+- Request and response examples that belong to every endpoint documentation have meaningful default values that follow a specific story.
+- Code examples and explanatory text are visually separated.
```

Comment:
Not sure if I get your point here. Sometimes, having an additional example right below the description in the spec helps to understand the property... So I would not force them to only have them in the examples section...

Reply:
I'm with @cgebken on this. The closer the example is to the definition that the example tries to demonstrate, the better.

Reply:
Good point. This was actually not related to the spec but to the visualization in the portal. I'll change that.